lumigator/schemas/lumigator_schemas/jobs.py

```diff
@@ -29,6 +29,7 @@ class JobStatus(LowercaseEnum):
     RUNNING = "running"
     FAILED = "failed"
     SUCCEEDED = "succeeded"
+    STOPPED = "stopped"
```

Comment:
My only question with this part, is does this impact any other logic? Like if logic is checking whether a job is either failed or succeeeded, they would now need to also check if it was stopped, since that's another valid end state.

Reply:
Yes, this is also a terminal state. I'll double check where do we need to check this, because iirc there _is_ a list of terminal states that is used for some decision.